addons/mail/static/src/components/thread_viewer/thread_viewer.xml

```diff
@@ -32,6 +32,7 @@
                         hasCurrentPartnerAvatar="props.hasComposerCurrentPartnerAvatar"
                         hasSendButton="props.hasComposerSendButton"
                         hasThreadTyping="props.hasComposerThreadTyping"
+                        isCompact="threadViewer.thread.model === 'mail.channel' ? !threadViewer.thread.mass_mailing: undefined"
```

Comment:
space before :

Reply:
But is it the right syntax?

Shouldn't it be `threadViewer.thread.model === 'mail.channel' && !threadViewer.thread.mass_mailing || undefined`

We want undefined to use default value if not `mail.channel`, but don't we also want undefined if not mailing channel, instead of forcing false?

Reply:
`X && Y || Z` is the same as `X ? Y : Z` for me, parentheses don't change anything I think as && has priority.
Anyway I see your point. I think the right syntax then should be something like : 
`threadViewer.thread.model === 'mail.channel' && threadViewer.thread.mass_mailing ? false : undefined`